src/styles/_calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  gap: 1px;
+  padding-top: 50px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+    background-color: #eee;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    @include square($cellSize);
+
+    transition-timing-function: linear;
+    transition-property: transform;
+    transition-duration: 0.3s;
```

You can do this with 1 line
also, create a variable for transition duration
```suggestion
    transition: transform  $transitionDuration linear;
```

---

got it